### ISSUE TITLE
Will there be support for fireforks?

### ISSUE THREAD

**Issue by CoolestGuy94** [AdvDebug/NoMoreCookies#20]
I am using LibreWolf, and I tried to see if it was supported, to which it wasn't. I wanted to know if there will be support for forks of Firefox. Thanks!

**Comment by AdvDebug** [comment]
not really, the tool works by checking if the process is signed by a browser publisher like mozilla, brave, yandex, etc. but LibreWolf and some other forks are not signed so it's hard to add support for them in the moment without breaking the tool security, i will find a way tho but i'm fixing the tool and making it depend on services instead of AppInit_DLLs cause it's useless when you have secure boot enabled and won't work on newer versions of windows so it will take time before i figure this out.
